scripts/uboot_helper

```diff
@@ -358,6 +358,73 @@ devices = \
         'rockchip_legacy_boot': '1'
       },
     },
+    'RK3568': {
+      'nanopi-r5c': {
+        'dtb': 'rk3568-nanopi-r5c.dtb',
+        'config': 'nanopi-r5c-rk3568_defconfig',
+        'soc': 'RK3568'
+      },
+      'nanopi-r5s': {
+        'dtb': 'rk3568-nanopi-r5s.dtb',
+        'config': 'nanopi-r5s-rk3568_defconfig',
+        'soc': 'RK3568'
+      },
+      'odroid-m1': {
+        'dtb': 'rk3568-odroid-m1.dtb',
+        'config': 'odroid-m1-rk3568_defconfig',
+        'soc': 'RK3568'
+      },
+      'quartz64-a': {
+        'dtb': 'rk3566-quartz64-a.dtb',
+        'config': 'quartz64-a-rk3566_defconfig',
+        'soc': 'RK3566'
+      },
+      'quartz64-b': {
+        'dtb': 'rk3566-quartz64-b.dtb',
+        'config': 'quartz64-b-rk3566_defconfig',
+        'soc': 'RK3566'
+      },
+      'rock-3a': {
+        'dtb': 'rk3568-rock-3a.dtb',
+        'config': 'rock-3a-rk3568_defconfig',
+        'soc': 'RK3568'
+      },
+      'rock-3c': {
+        'dtb': 'rk3568-rock-3c.dtb',
+        'config': 'rock-3c-rk3566_defconfig',
+        'soc': 'RK3566'
+      },
+    },
+    'RK3588': {
+      'nanopc-t6': {
```

Comment:
Are you going to distinguish T6 LTS vs T6 (a variant with LTE modem). It should be pritty much the same hardware. Are they going to use the same dtb file?

Reply:
Support for LTS variant was merged in U-Boot `v2025.01-rc1`, U-Boot will automatically set `fdtfile` to correct device tree depending on non-LTS vs LTS variant.

LibreELEC currently expect that a .dtb-file is explicitly pointed out in `extlinux.conf`, so this needs to be reworked to allow auto-load of correct dtb based on `fdtfile`.

Should be possible to use `fdtdir` or place .dtb-files in `/dtb/rockchip/<board>.dtb` or similar, and this require changes to upgrade/update scripts.

Reply:
> Should be possible to use `fdtdir` or place .dtb-files in `/dtb/rockchip/<board>.dtb` or similar, and this require changes to upgrade/update scripts.

AMLGX uses `fdtdir` to support the Odroid N2/N2+ boards. Have a look at the Amlogic mkimage.

Reply:
Have now included initial Rockchip support for using `FDTDIR /` in `extlinux.conf` and it seem to work okay.